### ISSUE TITLE
Loss function and Metric

### ISSUE THREAD

**Issue by Toledo1** [tongzheng1992/E-FCN#1]
Hello, thank you for making your source code available; I wish to know why you've not implemented the PU, UIoU metrics and ECE loss function as stated in the paper. Are they not more important for a good training comparing to CategoricalCrossentropy loss and accuracy metric used here?
